coralproject/talk
# [CORL-2308] Filter by deletedAt when retrieving users to delete

## What does this PR do?

Excludes users with `deletedAt` set when retrieving users to delete.

Users with `deletedAt` set are already deleted and do not need to be attempting a delete again.

## What changes to the GraphQL/Database Schema does this PR introduce?

None

## How do I test this PR?

- Delete a user using the delete flow
- Let it process the deletion (wait for job to run)
- Then wait again for job to run a second time
- Notice that you don't get a `user was already deleted` error
 
## How do we deploy this PR?

No special deploy notes.
